apps/gro/steps.js

```diff
@@ -152,7 +152,59 @@ module.exports = {
   '/confirm': {
     controller: require('./controllers/confirm'),
     template: 'confirm.html',
-    next: '/confirmation'
+    next: '/confirmation',
+    modelData: {
+      tableSections: [{
+        name: 'enquiry-details',
+        fields: [
+          'about-radio',
+          'type-radio',
+          'person-text',
+          'person-one',
+          'person-two',
+          'details-text',
+          'existing-radio',
+          'previous-radio',
+          'additional-text',
+          'additional-radio'
+        ]
+      }, {
+        name: 'contact-details',
+        fields: [
+          'name-text',
+          'email-text',
+          'post-text',
+          'country-text'
+        ]
+      }, {
+        name: 'order-details',
+        fields: [
+          'how-radio',
+          'online-toggle-text',
+          'telephone-toggle-text',
+          'post-toggle-text',
+          'which-radio',
+          'order-number-text',
+          'when-date'
+        ]
+      }],
+      modifiers: {
+        'post-text': function combineLinesOfAddress(values) {
+          var fields = [
+            'address-text-one',
+            'address-text-two',
+            'address-text-three',
+            'address-text-four',
+            'address-text-five'
+          ];
+          return _.chain(values)
+            .pick(fields)
+            .values()
+            .value()
+            .join('<br />');
+        }
+      }
+    }
```

Comment:
@JoeChapman did you have any thoughts about where this would be better placed? I think meta should be included in  steps-config to expose variables relevant to the step, page title etc, but I agree that the confirm table config could be better placed elsewhere, perhaps in a separate config file passed to the controller on initialisation. I noted on the https://github.com/UKHomeOffice/hof-controllers/pull/22 PR that I now think it would be better to have a confirm controller which extends BaseController and is used by default on confirm pages, the logic for exposing table data would be here rather than in the base controller